packages/lumx-core/src/scss/components/chip/_index.scss

```diff
@@ -111,7 +111,7 @@
     &.#{$lumx-base-prefix}-chip--color-light {
         @include lumx-state-as-selected(lumx-base-const("state", "DEFAULT"), lumx-base-const("theme", "DARK"));
 
-        &.#{$lumx-base-prefix}-chip--is-clickable:hover {
+        &.#{$lumx-base-prefix}-chip--is-clickable:hover:enabled {
```

Comment:
Same here, `:enabled` does not work with `<a>` used in Chip. 
This is testable in the "Clickable" story, when activating the dark background, the hover state does not work